driver/modern_bpf/CMakeLists.txt

```
@@ -12,6 +15,13 @@ else()
 endif()
 message(STATUS "${MODERN_BPF_LOG_PREFIX} MODERN_BPF_DEBUG_MODE: ${MODERN_BPF_DEBUG_MODE}")
 
+if(MODERN_BPF_COS_WORKAROUND)
+  set(COS_WORKAROUND "COS_WORKAROUND")
```

```suggestion
  message(STATUS "${MODERN_BPF_LOG_PREFIX} Enabling MODERN_BPF_COS_WORKAROUND")
```

And drop the else + message below :)